docs/getting-started/using-gnokey.md

```diff
@@ -0,0 +1,777 @@
+---
+id: using-gnokey
+---
+
+# Using `gnokey`
+
+## Overview
+In this tutorial, you will learn how to use the `gnokey` binary to interact with
+a Gno.land chain. You will learn how to create state-changing calls, run readonly
+queries without using gas, as well as create, sign, and broadcast airgapped
+transactions for full security.
+
+## Prerequisites
+- **`gno`, `gnokey`, and `gnodev` installed.** Reference the
```

Comment:
Formatting is off

Reply:
I've replaced these sections with only requiring `gnokey`. Please check out the files.